expression/builtin_op.go

```diff
@@ -739,6 +740,17 @@ func (c *unaryNotFunctionClass) getFunction(ctx sessionctx.Context, args []Expre
 	case types.ETInt:
 		sig = &builtinUnaryNotIntSig{bf}
 		sig.setPbCode(tipb.ScalarFuncSig_UnaryNotInt)
+	case types.ETJson:
```

Comment:
why not like the other type, add a new sig `builtinUnaryNotJSONSig`

Reply:
Good. I also agree add a new sig `builtinUnaryNotJSONSig` is better :handshake: